util/topgen.py

```diff
@@ -526,6 +527,15 @@ def generate_ac_range_check(topcfg: Dict[str, object], out_path: Path) -> None:
     ipgen_render("ac_range_check", topname, params, out_path)
 
 
+# Generate RACL collateral
+def generate_racl(topcfg: Dict[str, object], out_path: Path) -> None:
+    # Not all tops use RACL
+    if 'racl_config' not in topcfg:
+        return
```

Comment:
Wouldn't it make sense to set `topcfg['racl']` to `None` in this situation?

Reply:
The reason this was _not_ added is to make it easier to detect if no RACL is used, i.e., down below: `completecfg.get('racl', DEFAULT_RACL_CONFIG)`

Reply:
Detection of no RACL could be as  simple as `racl not in completecfg`, and we could have no trace of racl if it is not used. Perhaps the package could be avoided in those cases to keep it simple.